CHANGELOG-3.6.md

```diff
@@ -15,6 +15,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.5.0...v3.6.0).
 ### etcdctl v3
 
 - Add command to generate [shell completion](https://github.com/etcd-io/etcd/pull/13133).
+- Show db size in use when print endpoint status [Show db size in use](https://github.com/etcd-io/etcd/pull/13639)
```

Comment:
@kkkkun thanks for the quick update but you don't need to duplicate the `Show db size in use` text. Please keep only one mention, so something like, `When print endpoint status, [show db size in use](https://github.com/etcd-io/etcd/pull/13639)` or `[Show db size in use](https://github.com/etcd-io/etcd/pull/13639) when print endpoint status. Thanks!

Reply:
fixed! Thanks!